src/dex/uniswap-v3/constants.ts

```diff
@@ -0,0 +1,30 @@
+export const UNISWAPV3_QUOTE_GASLIMIT = 200_000;
```

Comment:
I just think this estimation is very off. In the case of using onchain query it was good because we capped gas limit we spend, but currently we don't have that restriction and this value may lead to very bad gas prices.
Maybe Jason (@ColonelJ ) has idea how to handle this part properly?

I just wanted to bring attention into this part and not let go undiscussed